### ISSUE TITLE
Clicking on Back/Forward arrows from browser deletes Filters added instead of navigating to corresponding page

### ISSUE THREAD

**Issue by lauramarcos** [chanzuckerberg/cryoet-data-portal#1081]
**Describe the bug**
 if you try adding in filters and clicking back, it removes the filters instead of bringing you to the previous page

**To Reproduce**
1. Go to https://cryoetdataportal.czscience.com/browse-data/datasets
2. Click on any filter from filters panel and add them
3. Click on <Back> on browser

**Screenshots**

https://github.com/user-attachments/assets/4f367837-2ad8-471d-b91b-e4eeaa9fba51

**Desktop (please complete the following information):**
 - OS: Sonoma 14.5
 - Browser:  Chrome

**Comment by kev-zunshiwang** [comment]
Yeah can we change the back button so it doesnt affect the filters? it should go to the previous page

**Comment by junxini** [comment]
sorry your comment got buried in my emails @codemonkey800, and Kevin's response resurfaced it. I agree with Kevin that it would go to the previous page instead of purely removing/adding filters

**Comment by kev-zunshiwang** [comment]
@codemonkey800 - apparently in kandarp's study, when people click into the download dialog and exit, then click the back button, it brings them into the dialog, is this the same issue? Will this PR fix also fix what's happening to the download dialog problem? @kandarpksk @junxini

**Comment by kev-zunshiwang** [comment]
yeah the back and forward button should be only about going to previous pages the user is on, never about going to the previous feature the user was on ---- could we do a bulk fix on where this all show up?
